When cross compiling in void, FIND_FILE fails to find the python
script used to configure SIP.

diff --git cmake/FindSIP.cmake cmake/FindSIP.cmake
--- cmake/FindSIP.cmake
+++ cmake/FindSIP.cmake
@@ -37,10 +37,8 @@ IF(SIP_VERSION)
   SET(SIP_FOUND TRUE)
 ELSE(SIP_VERSION)
 
-  FIND_FILE(_find_sip_py FindSIP.py PATHS ${CMAKE_MODULE_PATH})
-
   SET(ENV{PYTHONPATH} ${CMAKE_INSTALL_PREFIX}/${PYTHON_SITE_PACKAGES_DIR})
-  EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${_find_sip_py}
+  EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/cmake/FindSIP.py
                   OUTPUT_VARIABLE sip_config
                   RESULT_VARIABLE sip_config_returncode)
   IF(sip_config_returncode EQUAL 0)
